MangoPay/Libraries/RestTool.php

```diff
@@ -180,6 +192,7 @@ private function BuildRequest($urlMethod, $pagination, $additionalUrlParams = nu
 
         curl_setopt($this->_curlHandle, CURLOPT_CONNECTTIMEOUT, $this->GetCurlConnectionTimeout());
         curl_setopt($this->_curlHandle, CURLOPT_TIMEOUT, $this->GetCurlResponseTimeout());
+        curl_setopt($this->_curlHandle, CURLOPT_SSLVERSION, CURL_SSLVERSION_TLSv1_0);
```

Comment:
I'd rather wrap this call in a `if (getenv('TRAVIS'))` since it's not needed in production.